driver/csiplugin/controllerserver.go

```diff
@@ -2354,13 +2354,13 @@ func (cs *ScaleControllerServer) ControllerExpandVolume(ctx context.Context, req
 		glog.Errorf("unable to list quota for fileset [%v] in filesystem [%v]. Error [%v]", filesetName, filesystemName, err)
 		return nil, status.Error(codes.Internal, fmt.Sprintf("unable to list quota for fileset [%v] in filesystem [%v]. Error [%v]", filesetName, filesystemName, err))
 	}
-
+	// fmt.Printf("++++++++quota++++++++++%+v\n", quota)
```

Comment:
Remove commented logs if it is not required.

Reply:
removed commented comments used for debugging.